### PR TITLE
web-wallet: Add Create Wallet flow tests

### LINKED ISSUE #1443: Cover Create Wallet flow with tests
Currently, we don't have tests for the "Create Wallet" flow. Opening this issue to fill in the gap and add tests.

### PR DESCRIPTION
**Pull request by nortonandreev**
Resolves #1443

### CODE REVIEW

File: web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js [hunk 1]
```diff
@@ -1,18 +1,477 @@
...
+		// Ensure Next is enabled
+		expect(nextButton).toBeEnabled();
+	});
+
+	it("correctly renders the Mnemonic Preview page", async () => {
+		const { container, getByRole, getAllByRole } = render(Create);
+
+		await fireEvent.click(getByRole("button", { name: "Accept" }));
+
+		const firstCheckbox = getAllByRole("checkbox")[0];
```
[the remark this answers is not in the document]
Comment (nortonandreev) [reply web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js:135]: Yeah, I usually find the first way clear to read, but directly doing it within the `click` call saves a few lines of code. I don't have a strong preference, so I will instead change it the way you have suggested.

File: web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js [hunk 2]
```diff
@@ -1,18 +1,477 @@
...
+		await fireEvent.click(secondCheckbox);
+
+		await fireEvent.click(getByRole("button", { name: "Next" }));
+		await fireEvent.click(getByRole("button", { name: "Next" }));
+
+		const mnemonicSplit = mnemonic.split(" ");
+
+		// Iterates over the Mnemonic and selects each
+		// word in the correct order
+		// Note: This type of test won't work for mnemonic with repeated words
```
[the remark this answers is not in the document]
Comment (nortonandreev) [reply web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js:283]: Have removed the comment.

File: web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js [hunk 3]
```diff
@@ -1,18 +1,420 @@
...
+		await fireEvent.click(getAllByRole("checkbox")[0]);
+		await fireEvent.click(getAllByRole("checkbox")[1]);
+
+		await fireEvent.click(getByRole("button", { name: "Next" }));
+		await fireEvent.click(getByRole("button", { name: "Next" }));
+
+		const mnemonicSplit = mnemonic.split(" ");
+
+		mnemonicSplit.forEach(async word => {
+			const button = getByRole("button", { name: word });
```
Comment (deuch13) [inline web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js:222]: Missed this one

**Review by deuch13 — CHANGES_REQUESTED** [review]

File: web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js [hunk 4]
```diff
@@ -1,18 +1,477 @@
...
+	});
+
+	it("correctly renders the Mnemonic Preview page", async () => {
+		const { container, getByRole, getAllByRole } = render(Create);
+
+		await fireEvent.click(getByRole("button", { name: "Accept" }));
+
+		const firstCheckbox = getAllByRole("checkbox")[0];
+		const secondCheckbox = getAllByRole("checkbox")[1];
+		const nextButton = getByRole("button", { name: "Next" });
```
[the remark this answers is not in the document]
Comment (deuch13) [reply web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js]: Indeed, i was in a pasting spree

File: web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js (hunk 3 above)
Comment (nortonandreev) [reply web-wallet/src/routes/(welcome)/setup/create/__tests__/page.spec.js]: Done.

**Review by deuch13 — APPROVED** [review]

**Pull request merged**